### DISCUSSION TITLE
Support for other laguages written in non-latin letters

### DISCUSSION THREAD

**Discussion by rabio10** [ManimCommunity/manim#3425] [Ideas]
I am starting to learn using Manim for creating educational content about math, and it's pretty nice.
However, i thought about implementing some text in the animation that's is not an equation or something, and i tried Arabic letters , but  IT appears that's not supported.

**Comment by MrDiver** [comment]
It is supported? Did you use the Text class?

https://docs.manim.community/en/stable/reference/manim.mobject.text.text_mobject.Text.html?highlight=Text#multiplefonts

![image](https://docs.manim.community/en/stable/_images/MultipleFonts-1.png)
